docker/manage

```diff
@@ -444,7 +444,7 @@ configureEnvironment() {
   export SQL_DEBUG=${SQL_DEBUG:-}
   export WEB_CONCURRENCY=${WEB_CONCURRENCY:-5}
   export APPLICATION_URL=${APPLICATION_URL-http://localhost:8080}
-  export AGENT_ADMIN_API_KEY=${AGENT_ADMIN_API_KEY:-R2D2HfPM5Zwd69IjclQiuFmcMV6}
+  export AGENT_ADMIN_API_KEY=${AGENT_ADMIN_API_KEY}
```

Comment:
@nrempel, Assuming this is simply to make it easier to browse to the admin API during development; correct?

Reply:
Exactly. It's still possible to enable it for testing, but the goal is to make the default settings the most sane for new users